maintenance-mode.php

```diff
@@ -61,7 +159,11 @@ function current_user_can_bypass_vip_maintenance_mode() {
  * @since 0.1.1
  */
 function vip_maintenance_mode_template_redirect() {
-	if ( current_user_can_bypass_vip_maintenance_mode() ) {
+	if ( is_customize_preview() ) {
```

Comment:
Moving `is_customize_preview()` into a separate PR would no longer allow a user to deactivate the maintenance mode once it had been activated. The only way to deactivate the maintenance mode then would be to deactivate either the plugin or to alter the `vip_maintenance_mode_enable` setting within the options table. Therefore, I don't think that removing it would be in the interest of the user, unless I'm missing something here. 🤔